Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -1863,9 +1863,9 @@ WebIDL::ExceptionOr<void> HTMLInputElement::set_min_length(WebIDL::Long value)
 // https://html.spec.whatwg.org/multipage/input.html#the-size-attribute
 unsigned HTMLInputElement::size() const
 {
-    // The size IDL attribute is limited to only positive numbers and has a default value of 20.
+    // The size IDL attribute is limited to only numbers greater than zero and has a default value of 20.
```

Comment:
This comment is taken from the spec text and shouldn't be changed. Instead of changing this comment I would also include the spec text from the line above:
> The [size](https://html.spec.whatwg.org/multipage/input.html#attr-input-size) attribute, if specified, must have a value that is a [valid non-negative integer](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#valid-non-negative-integer) greater than zero.

Reply:
Oh, sorry about that! I searched for the comment in the HTML specs before making the change but missed it because I wasn’t looking in the right place (I found it here: https://html.spec.whatwg.org/multipage/input.html#attr-input-size).

I’ve now added the new comment from the specs before the old one, maintaining the same order as in the specs.

Thanks for pointing this out!